indico/modules/events/abstracts/models/call_for_abstracts.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.modules.events.abstracts.settings import abstracts_settings
+from indico.util.date_time import now_utc
+
+
+class CallForAbstracts(object):
+    """Proxy class to facilitate access to the call for abstracts settings"""
+
+    def __init__(self, event):
+        self.event = event
+        self.settings = abstracts_settings.get_all(self.event)
```

Comment:
There's lots of request-level caching on settings, so you could just `get()` them when needed instead of storing the dict here.